pkgs/node/src/chain.zig

```diff
@@ -359,8 +360,27 @@ pub const BeamChain = struct {
         }
 
         // 5. fc update head
-        _ = try self.forkChoice.updateHead();
+        const new_head = try self.forkChoice.updateHead();
         const processing_time = onblock_timer.observe();
+
+        // 6. Emit new head event via SSE
+        const metrics_proto_block = api.events.ProtoBlock{
```

Comment:
why is this called metrics proto block, and why do you need to redeclare proto block, move forkchoice's protoblock to types